packages/fast-development-site-react/src/components/shell/pane.tsx

```diff
@@ -27,8 +27,7 @@ const style: ComponentStyles<IShellPaneManagedClasses, IDevSiteDesignSystem> = {
         width: toPx(40),
     },
     shellPane__expanded: {
-        width: toPx(300),
-        padding: toPx(12)
+        width: toPx(300)
```

Comment:
Removing the padding would affect the form generator configuration style... is this accounted for?

Reply:
The form generator is using the same style...ok. I'll figure out a way to address this singular instance.

Reply:
Another option would be to bake this style into the form generator surrounding div or form element

Reply:
Proposal is up. Added an optional prop for className to enable styling within a layout so it doesn't have to be baked into the form itself.

Reply:
Cool looks good.